<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addScientistDialog</class>
 <widget class="QDialog" name="addScientistDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_add_new_scientist">
     <property name="text">
      <string>&lt;h2 style = 'color:blue'&gt;Add new scientist&lt;/h2&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_scientist_name">
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_error_scientist_name">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEdit_scientist_name"/>
   </item>
   <item>
    <widget class="QLabel" name="label_scientist_gender">
     <property name="text">
      <string>Gender</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="radioButton_scientist_male">
     <property name="text">
      <string>Male</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="radioButton_scientist_female">
     <property name="text">
      <string>Female</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_scientist_yearBorn">
       <property name="text">
        <string>Year born</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_error_yearBorn">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEdit_scientist_yearBorn"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_scientist_yearDeath">
       <property name="text">
        <string>Year of death</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_scientist_alive">
       <property name="text">
        <string>Alive</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_scientist_yearDeath_error">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEdit_scientist_yearDeath">
     <property name="enabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="Button_add_scientist">
     <property name="text">
      <string>Add scientist</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEdit_scientist_yearDeath</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
